acceptance/README.md

```diff
@@ -1,73 +1,97 @@
 of:
 
     curl --url vmpooler.delivery.puppetlabs.net/vm
 
-#### Define a hosts config file for your new VM
-
-Copy/modify the local RHEL-7 host config at
-`acceptance/config/beaker/jenkins/redhat7-64m-64a.cfg`  This file configures a
-basic 2-node configuration.
+## Install dependencies
 
-Please note, the most recent host configurations can be generated with the
-`beaker-hostgenerator` command available from the `beaker-hostgenerator` gem.
-Generating a new host configuration takes the form of `bundle exec beaker-hostgenerator redhat7-64ma`
+Beaker needs to be installed and is expressed as a Gemfile dependency.  To
+install beaker, simply run `bundle install --path .bundle/gems` from the top
+level directory of the project.
 
-The host expression is a bit strange and is parsed as per the documentation at
-`bundle exec beaker-hostgenerator --help`.  See also [the
-source][beaker-hostgenerator].
+Running beaker will then be a matter of using `bundle exec beaker ...` as described
+below.
 
-[beaker-hostgenerator]: https://github.com/puppetlabs/beaker-hostgenerator
+You might also need to obtain the `~/.ssh/id\_rsa-acceptance` file from a
+colleague so that beaker can SSH into the target VMs located in the pooler.
 
-#### Define the PACKAGE_BUILD_VERSION and PUPPET_VERSION environment variables
+### Define your test environment using environment variables
 
-You'll need to provide a couple environment variables that specify which build
-of puppet-server to install and test against.
+You'll need to provide two environment variables that specify which builds of the
+agent and server to install and test against, ```PACKAGE_BUILD_VERSION``` and
+`PUPPET_VERSION`. ```PACKAGE_BUILD_VERSION``` indicates
+the build of the puppet-server package to install.
 
-1. Go to http://builds.delivery.puppetlabs.net/puppetserver/
-2. Scroll down to the most recent build at the bottom.  This will look like:
-   `0.1.4.SNAPSHOT.2014.05.15T1118`
-3. Copy the text (not the link address) - this will be `PACKAGE_BUILD_VERSION`
-4. Define ```PUPPET_VERSION``` as the packaged version of the puppet-agent
-   package that we're depending upon, which is currently `1.1.0`
+1. Go to [http://builds.puppetlabs.lan/puppetserver/?C=M;O=D](http://builds.puppetlabs.lan/puppetserver/?C=M;O=D).
+1. This puts the most recent build at the top of the display. It will look like `2016.1.1004.SNAPSHOT.2016.02.03T1410`.
+1. This value (the text, not the link address) is ```PACKAGE_BUILD_VERSION```.
+1. `PUPPET_VERSION` specifies which version of the puppet-agent package to install.
+This is currently `1.3.4`.
 
 If you are testing backwards compaitiblity against a specific release of
-puppet, the `PUPPET_LEGACY_VERSION` environment variable is also available for
+puppet, the ```PUPPET_LEGACY_VERSION``` environment variable is also available for
 this purpose, but is not required.
 
-#### Install dependencies
+### Run Beaker from a script
```

Comment:
I humbly request that, because this is the most simple method of running beaker, that you move it up higher in the document.  But I don't insist on it; if you think it ought to stay where it's at I'll merge anyhow.